### PR TITLE
Update source_code.md

### PR DESCRIPTION
**Pull request by ggg27**
Add TOS;DR and Snowflake to Source_Code.md

### CODE REVIEW

File: source_code.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ https://github.com/privacytoolsIO/
  
  Decentraleyes: https://git.synz.io/Synzvato/decentraleyes
  
+ Terms of Service; Didn’t Read : https://github.com/tosdr/
+ 
+ Snowflake : https://gitweb.torproject.org/pluggable-transports/snowflake.git
```
Comment (Mikaela) [inline source_code.md]: I am not entirely sure this is the correct place.

**Comment by nitrohorse** [comment]
Thanks for catching this, @ggg27; didn't realize I need to update this file for changes.

File: source_code.md (hunk 1 above)
Comment (nitrohorse) [reply source_code.md]: I think this is the right place from what I can tell. It looks like the order matches the order on PTIO.

Comment (Mikaela) [reply source_code.md]: I mean is that the source code for the browser extension too?

Comment (nitrohorse) [reply source_code.md]: Oh gotcha; yeah this was the same link I found in the [docs](https://trac.torproject.org/projects/tor/wiki/doc/Snowflake).

Comment (nitrohorse) [reply source_code.md]: Wow I misunderstood again, sorry 😓 hmm I can’t seem to find the extension source code myself 🤔 Maybe it’s not open-source at the moment...

Comment (nitrohorse) [reply source_code.md]: This is the same URL in the Addons page so I think once the extension itself becomes open-source we should update both locations.

**Pull request merged**